redis/src/cluster_client.rs

```diff
@@ -210,11 +210,12 @@ impl ClusterClientBuilder {
         self
     }
 
-    /// Sets the default auto reconnect parameter for all new connections (default is true).
+    /// Disables auto reconnect for all new connections (default is enabled).
```

Comment:
Do we need to make this change? I admire your enthusiasm to clean stuff up, but let's keep our eyes on the lofty and noble goal of having a mainline async clustered redis client and minimize things like this :sweat_smile: